hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -365,6 +357,7 @@ public void addDependencies(ServiceTarget target, ServiceBuilder<MonitorService>
 
         // bind the API to JNDI so other apps can use it, and prepare to build the binder service
         // Note that if we are running in host controller or similiar, JNDI binding is not available.
+        /*
```

Comment:
It seems that JNDI stuff can also run on EAP6, it was only missing the ImmediateManagedReference class, which didn't seem hard to implement and use when the other isn't found. I'll push those changes later.